shared/gast_to_code/converter_registry.py

```diff
@@ -0,0 +1,12 @@
+
+class ConverterRegistry():
```

Comment:
Could add `get_langs()` to show in the UI.

Reply:
@g-harel already did this in language detection branch!